### DISCUSSION TITLE
✅ Bluethooth Keyboard not alle Keys are working well

### DISCUSSION THREAD

**Discussion by Supertas** [isontheline/pro.webssh.net#179] [General]
Hello,

thank you for the good ssh-Client. I am very happy with them.

But I have a small problem with a bluetooth keyboard.

I have problem with the page up and page down keys. They are not working. Looks like mostly all iOS ssh clients have here trouble.

I found only „ShellFish“ where I can use Pipe and  Page Up + Down.

Are you have any solution for this problem?

Greetings form Germany
Thomas

**Comment by isontheline** [comment]
Hello from France 👋

Thank you a lot for using WebSSH !

Perhaps there is a solution. 
I'm opening an issue in order to track my work : https://github.com/isontheline/pro.webssh.net/issues/180

Have a nice day ☀️

**Comment by isontheline** [comment]
Released inside [WebSSH 14.10 - Truck](https://webssh.net/documentation/changelog/14.10/)
